Gemfile

```diff
@@ -36,65 +36,65 @@ gem 'jbuilder', '~> 2.5'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+#My gems
 gem 'awesome_print'
 gem 'dotenv'
 gem 'dotenv-rails'
 gem 'httparty'
 gem 'nokogiri'
 gem 'puma_worker_killer'
-gem "rubycritic"
+gem 'rack-canonical-host'
 gem 'sassc-rails'
 gem 'sdoc',                 group: :doc
 gem 'slim'
 gem 'slim-rails'
 gem 'therubyracer',         platforms: :ruby
-gem 'rack-canonical-host'
+
 # assets
-gem "font-awesome-sass"
-gem "font-awesome-rails"
+gem 'font-awesome-sass'
+gem 'font-awesome-rails'
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `font-awesome-rails` should appear before `font-awesome-sass`.